rebasehelper/specfile.py

```diff
@@ -1271,13 +1270,27 @@ def get_requires(self):
         """
         return [r.decode(constants.DEFENC) if six.PY3 else r for r in self.hdr[rpm.RPMTAG_REQUIRES]]
 
-    def update_changelog(self):
-        """Inserts new entry into the changelog and saves the SpecFile."""
-        new_entry = self.get_new_log()
+    def update_changelog(self, changelog_entry):
```

Comment:
Yup, this is exactly what we were looking for.

Reply:
I am going to merge this. I may refactor the class even further (passing `download` argument to the constructor doesn't make sense for packit nor for rebase-helper). If I make any changes to the methods that packit uses, I will create a PR in packit to take the changes into account.

Reply:
Awesome! thank you.